st2actions/st2actions/runners/actionchainrunner.py

```diff
@@ -374,7 +374,10 @@ def _run_action(self, liveaction, wait_for_completion=True):
             liveaction, _ = action_service.request(liveaction)
         except:
             liveaction.status = LIVEACTION_STATUS_FAILED
-            raise Exception('Failed to schedule liveaction.')
+
+            message = 'Failed to schedule liveaction.'
+            LOG.exception(message)
```

Comment:
Yeah, I was actually thinking about that, but decided not to do it since we also don't do it in other places.

In any case, if you think that's better, I can make the change.